app/views/vacancies/index.html.erb

```diff
@@ -1,4 +1,17 @@
-<h1 class="page-title"><%= t('.title') %></h1>
+
+<section class="bg-vacancy-intro">
+  <div class="row">
+    <div class="col-md-12">
+      <h1 class="text-center">
+        Find the best job offers! <br>
+        Find the ideal professional for your company!
+      </h1>
+      <div class="text-center">
+          <button type="button" name="button" class="btn btn-primary btn-register-vacancy">Advertise for free</button>
```

Comment:
For me this should be a `link`, since it is a navigation flow. And this is not linking to nowhere. And this should be linking to `new_vacancy_path`.